conf/production.ini

```diff
@@ -15,6 +15,8 @@ use: egg:h
 #h.client_id:
 #h.client_secret:
 
+h.hashids.salt: production salt
```

Comment:
I'm inclined to remove this so that if you don't set the environment var in production the application will explode rather than silently using a predictable salt.